api/apps/api/src/modules/clone/export/adapters/export-adapters.module.ts

```diff
@@ -6,15 +6,25 @@ import { ExportRepository } from '../application/export-repository.port';
 import { ResourcePieces } from '../application/resource-pieces.port';
 import { ArchiveCreator } from '../application/archive-creator.port';
 import { ResourcePiecesAdapter } from './resource-pieces.adapter';
-import { InMemoryExportRepo } from './in-memory-export.repository';
 import { NodeArchiveCreator } from './node-archive-creator';
+import { TypeormExportRepository } from '@marxan-api/modules/clone/export/adapters/typeorm-export.repository';
+import { ExportEntity } from '@marxan-api/modules/clone/export/adapters/entities/exports.api.entity';
+import { ExportComponentEntity } from '@marxan-api/modules/clone/export/adapters/entities/export-components.api.entity';
+import { ComponentLocationEntity } from '@marxan-api/modules/clone/export/adapters/entities/component-locations.api.entity';
 
 @Module({
-  imports: [FileRepositoryModule, TypeOrmModule.forFeature([])],
+  imports: [
+    FileRepositoryModule,
+    TypeOrmModule.forFeature([
+      ExportEntity,
+      ExportComponentEntity,
+      ComponentLocationEntity,
+    ]),
+  ],
   providers: [
     {
       provide: ExportRepository,
-      useClass: InMemoryExportRepo, // TODO TypeormExportRepository
```

Comment:
This seems like the last usage of the InMemory repo besides tests, so we should probably get rid of it.
A related question: why is ExportRepository an abstract class instead of an interface?

Reply:
In order to be able to use `ExportRepository` as the `provide` property of the provider. We can't use a Typescript interface as a value for `provide` property. We could also declare it as an interface and use a Symbol for the `provide` property along as we use the `@Inject` decorator for DI. `TypeormExportRepository` and `InMemoryExportRepo` implements the abstract class, so we should be safe 🤔